app/views/i18n/I18NEditModelView.coffee

```diff
@@ -128,54 +140,55 @@ module.exports = class I18NEditModelView extends RootView
     #- Enable patch submit button
 
     @$el.find('#patch-submit').attr('disabled', null)
+    @madeChanges = true
```

Comment:
I might remove the newlines immediately after each line comment and at the beginning of the function for readability. This feels a bit too spacious.